tests/gamma/csm_observability_test.py

```diff
@@ -454,6 +470,20 @@ def build_counter_query(cls, metric_type: str, namespace: str) -> str:
             f'resource.labels.namespace = "{namespace}"'
         )
 
+    @staticmethod
+    def filter_label_based_on_lang(
+        language: _Lang, labels: Dict[str, Any]
+    ) -> None:
+        """
+        Filter labels to check based on language.
+        """
+        if language == _Lang.PYTHON:
+            # TODO(xuanwn): Remove this once https://github.com/open-telemetry/opentelemetry-python/issues/3072 is fixed.
+            if "otel_scope_version" in labels:
+                del labels["otel_scope_version"]
+            if "otel_scope_name" in labels:
+                del labels["otel_scope_name"]
```

Comment:
optional: this can be simplified a bit
```suggestion
            labels.pop("otel_scope_version", None)
            labels.pop("otel_scope_name", None)
```